libs/cases/frontend/view/src/lib/components/cases-browse/case-browse.component.ts

```diff
@@ -2,7 +2,9 @@ import { CommonModule } from '@angular/common';
 import { ChangeDetectionStrategy, Component, computed, inject, ViewEncapsulation } from '@angular/core';
 import { toSignal } from '@angular/core/rxjs-interop';
 import { RouterLink } from '@angular/router';
+// eslint-disable-next-line @nx/enforce-module-boundaries
```

Comment:
Please don't add eslint disable. I think since it is not on develop it should not me necessary anyways. 

Same for all the other eslint disables

Reply:
Ok, has been done :)

Reply:
Thanks, I approved it now